go/vt/vtgate/executorcontext/safe_session.go

```diff
@@ -390,6 +390,18 @@ func (session *SafeSession) GetCommitOrder() vtgatepb.CommitOrder {
 	return session.commitOrder
 }
 
+func (session *SafeSession) SetTxErrorBlockNextQueries(enable bool) {
+	session.mu.Lock()
+	defer session.mu.Unlock()
+	session.TxErrorBlockNextQueries = enable
+}
+
+func (session *SafeSession) GetTxErrorBlockNextQueries() bool {
+	session.mu.Lock()
+	defer session.mu.Unlock()
+	return session.Session.GetTxErrorBlockNextQueries()
```

Comment:
nit: for boolean type, we can name the method with `Is`

Reply:
Fixed via 812df8f0dd6